stages/apps/sharedsvc/variables.tf

```diff
@@ -54,4 +54,31 @@ variable "vault_address" {
   default = "https://vault.internal.services.liatr.io"
 }
 
+variable "github_runners_service_account_arn" {
+}
+
 variable "iam_caller_identity_headers" {}
+
+# example_value = {
```

Comment:
Is there a reason we left these comments in?

Reply:
The object structure is somewhat complex and not immediately obvious; it would be nice if we could make it so the comments aren't needed, I agree.

Reply:
note: tried converting the complex map to a set of objects, but terraform's `for_each` only supports maps or sets of strings